scripts/CMSRunAnalysis.py

```diff
@@ -18,16 +18,36 @@
 import logging
 import commands
 import traceback
-import subprocess
 from ast import literal_eval
 from optparse import OptionParser, BadOptionError, AmbiguousOptionError
 
 import DashboardAPI
 import WMCore.Storage.SiteLocalConfig as SiteLocalConfig
 
+# replicate here code from ServerUtilities.py to avoid importing CRABServer in jobs
+# see there for more documentation. Ideally could move this to WMCore
+class tempSetLogLevel():
+    """
+        a simple context manager to temporarely change logging level
+        USAGE:
+            with tempSetLogLevel(logger=myLogger,level=logging.ERROR):
+               do stuff
+    """
+    import logging
```

Comment:
Already in line [18](https://github.com/dmwm/CRABServer/pull/5887/files#diff-b4df3d7055437f8d2eeafa0b449218d5R18)

Reply:
OK